### DISCUSSION TITLE
max-height

### DISCUSSION THREAD

**Discussion by amery** [beercss/beercss#118] [Q&A] [answered]
Hi, to inaugurate this lovely board, I wanted to ask for an example of how to make a card/article in the middle of the screen 😅
filling the whole canvas in background colour

**Comment by laurentpayot** [comment, marked as answer]
Hi @amery. Is the [layout section of the docs](https://www.beercss.com/#layouts) what you are looking for?

**Reply by amery** [reply]
My problem with what I've tried is that the body ends half way, only occupying the height of the card

**Reply by leonardorafael** [reply]
Sorry I don't understand. Do you want something like this https://codepen.io/leo-bnu/pen/eYjGBwY?

![image](https://user-images.githubusercontent.com/1918010/212884179-255e682c-2653-4839-bd4b-d6a7cf70ddc2.png)

**Reply by amery** [reply]
thanks! `article.absolute` made the trick :D

https://licensly.jpi.workers.dev/
